### ISSUE TITLE
Published datasets returning to "In Progress" status

### ISSUE THREAD

**Issue by jleighherzog** [datadryad/dryad-product-roadmap#3326]
Priority: Medium

Kailasha shared [this link](https://datadryad.org/stash/ds_admin?tenant=&editor_id=86101&curation_status=in_progress&publication_name=&sponsor_org=&direction=asc&sort=updated_at) to her dashboard which shows 3 published datasets that were returned to "In Progress" status. 

Any idea why this might be happening? Kailasha confirmed that she didn't attempt to access these submissions post-pub

In [this example](https://datadryad.org/stash/ds_admin/activity_log/112520?), it looks like AAAS Admin made a status change. Do they have permissions to do that?

**Comment by ahamelers** [comment]
> In [this example](https://datadryad.org/stash/ds_admin/activity_log/112520?), it looks like AAAS Admin made a status change. Do they have permissions to do that?

Yes (#2701 tracks the history of this saga)

**Comment by jleighherzog** [comment]
@ahamelers : Ok, will close this ticket and proceed with guidance provided in #2701

**Comment by jleighherzog** [comment]
Update from Kailasha: "I think we have an answer to this. An author contacted the helpdesk ([SF 33322](https://dryad.lightning.force.com/lightning/r/Case/500Pl00000C27coIAB/view)) regarding a published dataset that went into 'In Progress' status. 

I checked the activity log. Although the author has mentioned that they went into editing mode and then abruptly ended the session, the [activity lo](https://datadryad.org/stash/ds_admin/activity_log/99023?)g does not show their name. It shows my name. So, I guess this is happening with only those published datasets which were accessed by the authors who abruptly closed the window without hitting 'Cancel and discard changes' or 'Submit'. When I clicked 'Forcibly edit dataset' and then selected 'Cancel and discard changes' the dataset status reverted back to the previous status 'Published' with the previous publication date."
